sound/soc/sof/intel/hda-loader.c

```diff
@@ -496,6 +496,10 @@ int hda_dsp_ext_man_get_cavs_config_data(struct snd_sof_dev *sdev,
 			hda->clk_config_lpro = config_data->elems[i].value;
 			dev_dbg(sdev->dev, "FW clock config: %s\n", hda->clk_config_lpro ? "LPRO" : "HPRO");
 			break;
+		case SOF_EXT_MAN_CAVS_CONFIG_OUTBOX_SIZE:
+		case SOF_EXT_MAN_CAVS_CONFIG_INBOX_SIZE:
+			/* This elem is supported but config data is not being used yet */
```

Comment:
what does supported mean and who defines this?

Reply:
I didn't like this phase either but I copied from merged commit to get less review feedback.  I re-phased with what I feel right description